images/base/Dockerfile

```diff
@@ -9,3 +9,7 @@ FROM centos:centos7
 # components from EPEL must be installed in a separate yum install step
 RUN yum install -y git tar wget socat hostname sysvinit-tools util-linux epel-release && \
     yum clean all
+
+ADD persistent_volume_scrubber /usr/local/bin/
```

Comment:
You should create a script directory and then add the entire script directory.  The script directory should be under the root `scripts/volumes` and the filename should be `recycle.sh`.  You should add `scripts` into `/usr/share/openshift` or whatever the appropriate path location is for installing client specific scripts.  You shouldn't need to chmod if the file in source control is properly chmodded.